ads/adpicker.md

```diff
@@ -0,0 +1,38 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law o
+r agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AdPicker
+
+Please visit our website (https://adpicker.net) for more details
+
+## Example
+
+```html
+<amp-ad width="300" height="250"
+    type="adpicker"
+    data-ph="kbj2l87p3j">
+</amp-ad>
+```
+
+## Configuration
+
+Supported parameters:
+
+**Required**
+- width:        required by amp
```

Comment:
FYI: It is not always the case that AMP require both `width` and `height`. Please refer to supported amp-ad doc [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/amp-ad.md#-amp-ad--amp-embed). 
Please only specify parameters required by your ad network here. If you want to add instructions on how to use the `<amp-ad>`, you can refer to our README here.